hack/update-bazel.sh

```diff
@@ -22,5 +22,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 git config http.https://gopkg.in.followRedirects true
 
+# Remove generated files prior to running gazel.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
Can we maybe use +build tags to make bazel ignore it or something?  Or can it come with a TODO comment to remove this line completely once Bazel is the only way to build?